### PR TITLE
parse 400g zr port config from minigraph

### PR DESCRIPTION
**Pull request by arlakshm**
Signed-off-by: Arvindsrinivasan Lakshmi Narasimhan <arlakshm@microsoft.com>

#### Why I did it
Generate the port configuration required 400G ZR port from minigraph.

#### How I did it
Add parse logic to get `tx_power` and `laser_freq` from `LinkMetadata` section of the minigraph.
Add UT for packet-chassis and voq chassis

#### How to verify it
UT

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -1003,6 +1008,11 @@ def parse_linkmeta(meta, hname):
             linkmetas[port]["AutoNegotiation"] = auto_negotiation
         if macsec_enabled:
             linkmetas[port]["MacSecEnabled"] = macsec_enabled
+        if tx_power:
+            linkmetas[port]["tx_power"] = tx_power
```
Comment (prgeor) [inline src/sonic-config-engine/minigraph.py:1012]: tx_power can be float, so is that taken care?

File: src/sonic-config-engine/tests/sample-voq-graph.xml [hunk 2]
```diff
@@ -79,6 +79,20 @@
...
+      <a:LinkMetadata>
+        <a:Name i:nil="true"/>
+        <a:Properties>
+          <a:DeviceProperty>
+            <a:Name>Frequency</a:Name>
+            <a:Value>195.875</a:Value>
+          </a:DeviceProperty>
+          <a:DeviceProperty>
+            <a:Name>TxPower</a:Name>
+            <a:Value>-10</a:Value>
```
Comment (prgeor) [inline src/sonic-config-engine/tests/sample-voq-graph.xml:91]: can we have one test case for float value? like -10.3? see https://github.com/sonic-net/sonic-buildimage/pull/11053

File: src/sonic-config-engine/minigraph.py (hunk 1 above)
Comment (arlakshm) [reply src/sonic-config-engine/minigraph.py:1012]: Yes, this is taken care

File: src/sonic-config-engine/tests/sample-voq-graph.xml (hunk 2 above)
Comment (arlakshm) [reply src/sonic-config-engine/tests/sample-voq-graph.xml:91]: updated the test to test this

**Review by prgeor — APPROVED** [review]

**Pull request merged**